Resources/Prototypes/_NF/Datasets/Names/cat_cult.yml

```diff
@@ -0,0 +1,12 @@
+- type: dataset # Names of town that held witch trials
```

Comment:
For your perusal: https://en.wikipedia.org/wiki/List_of_people_executed_for_witchcraft#List_of_people_executed_for_witchcraft

Glamis, Waterhouse, Leatherlips, Wardwell and maybe "Soulmother" seem like interesting names to me.